### PR TITLE
Initial update to 1.11 Forge.

### PR DESCRIPTION
**Pull request by kenzierocks**
Tested:
- [x] `//set`, `//undo`, `//redo` -- the basics
- [x] `//regen` -- this broke last time
- [x] Tile entities

Also updates Gradle to 3.2.1, because keeping updated is good.

### CODE REVIEW

**Comment by kenzierocks** [comment]
`//regen` seems to be working as before, but I'm going to rewrite it to work better (i.e. not generate trees all random-like)

**Comment by kenzierocks** [comment]
This is now ready for review @wizjany @DarkArc.

File: worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java [hunk 1]
```diff
@@ -243,73 +266,41 @@ public boolean regenerate(Region region, EditSession editSession) {
...
+        try {
+            for (BlockVector vec : region) {
+                editSession.setBlock(vec, from.getBlock(vec));
             }
+        } catch (MaxChangedBlocksException e) {
+            throw new RuntimeException(e);
+        } finally {
+            saveFolder.delete();
+            DimensionManager.setWorld(originalWorld.provider.getDimension(), null, server);
+            DimensionManager.setWorld(originalWorld.provider.getDimension(), originalWorld, server);
```
Comment (DarkArc) [inline worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java]: Why does this need nulled first?

File: worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java [hunk 2]
```diff
@@ -243,73 +266,41 @@ public boolean regenerate(Region region, EditSession editSession) {
...
         }
 
-        return false;
+        return true;
```
Comment (DarkArc) [inline worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java]: :+1: on rewriting this method

File: worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java (hunk 1 above)
Comment (kenzierocks) [inline worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java]: I'm not sure it needs to be, but it triggers the unload handlers this way.

**Pull request merged**